www/class/centreon-clapi/centreonACL.class.php

```diff
@@ -69,7 +69,8 @@ public function reload($flagOnly = false)
         $this->db->query("UPDATE acl_groups SET acl_group_changed = 1");
         $this->db->query("UPDATE acl_resources SET changed = 1");
         if ($flagOnly == false) {
-            passthru(CentreonUtils::getCentreonPath() . '/cron/centAcl.php');
+            $centreonDir = realpath(__DIR__ . "/../../../");
```

Comment:
```suggestion
```

Reply:
$this->centreon_path is not a property of this class